v2/kafka-to-pubsub/src/main/resources/kafka_to_pubsub_metadata.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "Kafka to Pub/Sub",
+  "description": "Streaming pipeline. Ingests JSON-encoded messages from Kafka to a Pub/Sub topic",
+  "parameters": [
+    {
+      "name": "bootstrapServers",
+      "label": "Kafka Bootstrap Servers",
+      "helpText": "Comma separated kafka bootstrap servers in format ip:port",
+      "regexes": [],
+      "paramType": "TEXT"
+    },
+    {
+      "name": "inputTopics",
+      "label": "Kafka input topics",
+      "helpText": "Comma separated Kafka topics to read data",
+      "regexes": [],
+      "paramType": "TEXT"
+    },
+    {
+      "name": "outputTopic",
+      "label": "Pub/Sub output topic",
+      "helpText": "Pub/Sub topic to write the output, in the format of 'projects/yourproject/topics/yourtopic'",
+      "regexes": [
+        "^projects\\/[^\\n\\r\\/]+\\/topics\\/[^\\n\\r\\/]+$"
+      ],
+      "paramType": "TEXT"
+    },
+    {
+      "name": "outputFormat",
+      "label": "Message format that is written into Pub/Sub",
+      "helpText": "One of two message formats that is written to Pub/Sub. JSON or AVRO",
+      "regexes": [
+        "^(JSON|AVRO)$"
+      ],
+      "isOptional": true
+    },
+    {
+      "name": "javascriptTextTransformGcsPath",
+      "label": "GCS location of your JavaScript UDF",
+      "helpText": "The full URL of your .js file. Example: gs://your-bucket/your-function.js",
+      "isOptional": true,
+      "regexes": [
+        "^gs:\\/\\/[^\\n\\r]+$"
+      ],
+      "paramType": "GCS_READ_FILE"
+    },
+    {
+      "name": "javascriptTextTransformFunctionName",
+      "label": "The name of the JavaScript function you wish to call as your UDF",
+      "helpText": "The function name should only contain letters, digits and underscores. Example: 'transform' or 'transform_udf1'.",
+      "isOptional": true,
+      "regexes": [
+        "[a-zA-Z0-9_]+"
+      ],
+      "paramType": "TEXT"
+    },
+    {
+      "name": "outputDeadLetterTopic",
+      "label": "Topic for messages failed to reach the output topic(aka. DeadLetter topic)",
+      "helpText": "Messages failed to reach the output topic for all kind of reasons (e.g., mismatched schema, malformed json) are written to this table. If it doesn't exist, it will be created during pipeline execution. Ex: 'projects/yourproject/topics/yourtopic'",
```

Comment:
Incorrect help text as the text refers to the BigQuery deadletter table used in other examples. In current Pipeline failed messages while applying javascript Transformation are sent to Pub/Sub topic

Reply:
Fixed the help text.